bundles/org.openhab.core.config.core/src/main/java/org/openhab/core/config/core/dto/ConfigDescriptionParameterDTO.java

```diff
@@ -29,6 +30,7 @@
 public class ConfigDescriptionParameterDTO {
 
     public String context;
+    @SerializedName(value = "default")
```

Comment:
As mentioned in https://github.com/openhab/openhab-core/pull/2376#discussion_r637870924 we can use the following syntax to allow both names for parsing.

```suggestion
    @SerializedName(value="default", alternate="defaultValue")
```

Reply:
Sounds like the safer option in any case. Shall we use this one? What is your preference?

Reply:
I'd prefer adding the fallback.